src/averageAges.js

```diff
@@ -15,11 +16,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = century
```

Comment:
You can make this more concise and efficient by combining the conditions in a single filter operation. Here's how you could refactor it:
const men = people.filter((p) => p.sex === 'm' && (!century || Math.ceil(p.died / 100) === century));

This approach ensures that we iterate over the people array only once and checks both conditions (sex and century) in the same iteration. It streamlines the code and might offer slight performance benefits, especially for larger datasets.